bin/pre-commit

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-PATH="$PATH:/usr/local/bin"
```

Comment:
@nosolosw For those of us using GUI git apps like Tower.app, removing PATH causes commit to stop working as the app no longer has PATH info to run the commands in `.git/hooks/pre-commit`. Can you revert this line?

More info: https://www.git-tower.com/help/mac/faq-and-tips/faq#faq-hook-scripts